### PR TITLE
Add new Dockerfiles for centos and rhel7 that fit the standard shape

### PR DESCRIPTION
**Pull request by smarterclayton**
Remove building with RPM in these images.  Changes already made to openshift/release

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/image-registry/_output/local/bin/linux/amd64/dockerregistry /usr/bin/
+ENV REGISTRY_CONFIGURATION_PATH=/config.yml
+COPY images/dockerregistry/config.yml ${REGISTRY_CONFIGURATION_PATH}
+USER 1001
+EXPOSE 5000
+VOLUME /registry
+CMD ["/usr/bin/dockerregistry", "${REGISTRY_CONFIGURATION_PATH}"]
+LABEL io.k8s.display-name="OpenShift Image Registry" \
+      io.k8s.description="This is a component of OpenShift and exposes a Docker registry that is integrated with the cluster for authentication and management." \
```
[the remark this answers is not in the document]
Comment (smarterclayton) [reply Dockerfile]: Two for the price of one - `a container image registry`

**Comment by smarterclayton** [comment]
/hold

Actual break, probably related to the cmd or the placement of the user stanza

**Comment by bparees** [comment]
The jobs are already updated to use these dockerfiles?

**Comment by smarterclayton** [comment]
This is failing for the same reason that all gcp runs are failing - the pull errors against docker hub.  will leave hold until we resolve that

**Comment by smarterclayton** [comment]
Removing hold

**Pull request merged**